app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -11,4 +17,20 @@ module TwoFactorAuthenticatable
     before_action :apply_secure_headers_override, only: %i[show create]
     # rubocop:enable Rails/LexicallyScopedActionFilter
   end
+
+  def self.direct_otp_length
+    OTP_LENGTH
+  end
+
+  def self.otp_length
+    OTP_LENGTH
+  end
+
+  def self.direct_otp_valid_for_seconds
+    DIRECT_OTP_VALID_FOR_SECONDS
+  end
+
+  def self.allowed_otp_drift_seconds
+    ALLOWED_OTP_DRIFT_SECONDS
+  end
```

Comment:
why define methods, and not access them as constants?

Reply:
Some of the specs stubbed methods to a return different config values, so I maintained the format, but I'm open to changing both to use constants

Reply:
if it's not too many specs, could be worth it! just to remove some indirection

Reply:
done in b1c3b58